.jenkins/cscs/env-gcc-oldest.sh

```diff
@@ -23,15 +23,14 @@ module load daint-mc
 spack load cmake@3.18.6
 spack load ninja@1.10.0
 
-configure_extra_options+=" -DHPX_WITH_MAX_CPU_COUNT=128"
-configure_extra_options+=" -DHPX_WITH_MALLOC=system"
-configure_extra_options+=" -DHPX_WITH_FETCH_ASIO=ON"
-configure_extra_options+=" -DHPX_WITH_CXX${CXX_STD}=ON"
-configure_extra_options+=" -DHPX_WITH_GENERIC_CONTEXT_COROUTINES=ON"
+configure_extra_options+=" -DHPXLocal_WITH_MAX_CPU_COUNT=128"
+configure_extra_options+=" -DHPXLocal_WITH_MALLOC=system"
+configure_extra_options+=" -DHPXLocal_WITH_CXX${CXX_STD}=ON"
+configure_extra_options+=" -DHPXLocal_WITH_GENERIC_CONTEXT_COROUTINES=ON"
+configure_extra_options+=" -DHPXLocal_WITH_APEX=ON"
```

Comment:
```suggestion
configure_extra_options+=" -DHPX_WITH_APEX=ON"
```
? cause in the env-clang-apex.sh, this one is stated